scrapers/OldGoesYoung.yml

```diff
@@ -0,0 +1,24 @@
+name: OldGoesYoung
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - oldgoesyoung.com/episode/
+      - trickyoldteacher.com/episode/
+      - spoiledvirgins.com/episode/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="wrapper"]/div[@class="set-block-wrapper"]/h1/text()
+      Details: //div[@class="wrapper"]/div[@class="set-block-wrapper"]/div[@class="set-bg"]/p[@class="set-description"]/text()
+      Date: 
+        selector: //div[@class="wrapper"]/div[@class="set-block-wrapper"]/div[@class="set-bg"]/div[@class="set-info"]/p[@class="set-info-stats"]/span()
```

Comment:
this selector matches also the length and rating even the `|`
you need to limit that eg 
`selector: //div[@class="wrapper"]/div[@class="set-block-wrapper"]/div[@class="set-bg"]/div[@class="set-info"]/p[@class="set-info-stats"]/span[1]